charmap.txt

```diff
@@ -414,7 +414,7 @@ HIGHLIGHT = FC 02 @ same as fc 01
 SHADOW = FC 03 @ same as fc 01
 COLOR_HIGHLIGHT_SHADOW = FC 04 @ takes 3 bytes
 PALETTE = FC 05 @ used in credits
-SIZE = FC 06 @ note that anything other than "SMALL" is invalid
+FONT = FC 06 @ valid values are 0, 1, 2, 7 and 8
```

Comment:
Oops, sorry to not lump the comments together, but looks like 6 is a valid font ID as well (for Braille)

Reply:
Should I modify the comment like "0, 1, 2, 6, 7 and 8" or "0, 1, 2, 6 (braille), 7 and 8"? :eyes:

Reply:
Sure. If we end up adding font id constants later we'll probably replace those numbers with the constant names, or a note on where to find them.

Reply:
> Whoops by sure I meant the latter lol

Done :eyes: